### PR TITLE
Add code of conduct 

### PR DESCRIPTION
**Pull request by neeteshking21**

### CODE REVIEW

**Comment by mohan-murali** [comment]
Create a new branch and rebase it from main.

**Review by NS007-dev — CHANGES_REQUESTED** [review]
~@Neeteshking21 do we need the .docker files? Or can they be removed from the commit?~

**Comment by NS007-dev** [comment]
Sorry for the above comment, I didn't see the other commits.

**Comment by eddiejaoude** [comment]
Not sure why there are so many other changes in this PR, should only be the code of conduct?

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Too many changes, please only include the Code of Conduct

**Comment by EmmaDawsonDev** [comment]
Do you need any help with this @Neeteshking21 ?

**Comment by neeteshking21** [comment]
Yes, off course.@emmalearnscode

File: README.md [hunk 1]
```diff
@@ -67,4 +67,4 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/d
 
 ### `npm run build` fails to minify
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
```
Comment (suitupstan) [inline README.md:70]: Please revert this change

**Review by suitupstan — CHANGES_REQUESTED** [review]
Please only add code of conduct, move the rest changes to another PR 🙂

**Comment by EmmaDawsonDev** [comment]
Thank you for contributing @Neeteshking21 I see someone else already added a code of conduct to the project and since this pr currently has lots of conflicts I will close it but please feel free to make more PRs in the future :)

**Pull request closed without merging**